cli/azd/cmd/root.go

```diff
@@ -322,19 +322,24 @@ func NewRootCmd(ctx context.Context, staticHelp bool, middlewareChain []*actions
 			return !descriptor.Options.DisableTelemetry
 		})
 
-	// Register common dependencies for the IoC container
-	ioc.RegisterInstance(ioc.Global, ctx)
```

Comment:
Can we delete the `Global` export from the `ioc` package as well?  It is no longer in use.